tests/conftest.py

```diff
@@ -23,18 +23,52 @@
 )
 from ophyd_async.epics import adsimdetector
 
-PANDA_RECORD = str(Path(__file__).parent / "fastcs" / "panda" / "db" / "panda.db")
+PANDA_RECORD = str(
+    Path(__file__).parent / "unit_tests" / "fastcs" / "panda" / "db" / "panda.db"
+)
 INCOMPLETE_BLOCK_RECORD = str(
-    Path(__file__).parent / "fastcs" / "panda" / "db" / "incomplete_block_panda.db"
+    Path(__file__).parent
+    / "unit_tests"
+    / "fastcs"
+    / "panda"
+    / "db"
+    / "incomplete_block_panda.db"
 )
 INCOMPLETE_RECORD = str(
-    Path(__file__).parent / "fastcs" / "panda" / "db" / "incomplete_panda.db"
+    Path(__file__).parent
+    / "unit_tests"
+    / "fastcs"
+    / "panda"
+    / "db"
+    / "incomplete_panda.db"
 )
 EXTRA_BLOCKS_RECORD = str(
-    Path(__file__).parent / "fastcs" / "panda" / "db" / "extra_blocks_panda.db"
+    Path(__file__).parent
+    / "unit_tests"
+    / "fastcs"
+    / "panda"
+    / "db"
+    / "extra_blocks_panda.db"
 )
 
 
+def pytest_configure(config):
+    config.addinivalue_line(
+        "markers",
+        "skip_as_not_ready: mark test to be skipped as not ready for general use",
+    )
+
+
+def pytest_collection_modifyitems(config, items):
+    for item in items:
+        if "skip_as_not_ready" in item.keywords:
+            item.add_marker(
+                pytest.mark.skip(
+                    reason="This test is skipped as not ready for general use"
+                )
+            )
```

Comment:
```suggestion
```

I believe if we make use of `@pytest.mark.skip(reason="")` later we do not need these magics?